tests/test_resources/test_envs/test_env.py

```diff
@@ -199,3 +199,35 @@ def test_working_dir_env(self, env, cluster, tmp_path):
         assert working_dir.name not in cluster.run(["ls"])[0][1]
 
         _uninstall_env(env, cluster)
+
+    @pytest.mark.level("minimal")
+    def test_secrets_env(self, env, cluster):
+        path_secret = rh.provider_secret(
+            "lambda", values={"api_key": "test_api_key"}
+        ).write(path="~/lambda_keys")
+        api_key_secret = rh.provider_secret(
+            "openai", values={"api_key": "test_openai_key"}
+        )
+        named_secret = (
+            rh.provider_secret("huggingface", values={"token": "test_hf_token"})
+            .write(path="~/hf_token")
+            .save()
+        )
+        secrets = [path_secret, api_key_secret, named_secret.provider]
```

Comment:
Can we add one which is just the string name for a builtin provider?

Reply:
that's what I'm trying to achieve with the named secret, just passing in `named_secret.provider` as a string. constructed a resource for it for the test to stay consistent, rather than needing to depend on a user's env setup or overriding any of the actual default env vars or paths.

Reply:
Oh oh I didn't get that, cool!